README.md

```diff
@@ -7,6 +7,9 @@
 [![All Contributors](https://img.shields.io/badge/all_contributors-10-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
+<img src="https://github.com/quarkiverse/quarkus-quinoa/blob/main/docs/modules/ROOT/assets/images/quarkus.svg" height="50" ><img src="https://github.com/quarkiverse/quarkus-quinoa/blob/main/docs/modules/ROOT/assets/images/plus-sign.svg" height="50" ><img src="https://github.com/quarkiverse/quarkus-quinoa/blob/main/docs/modules/ROOT/assets/images/node-logo.svg" height="50" >
```

Comment:
image does not appear in README.md

![image](https://user-images.githubusercontent.com/91419219/232323999-f37d7581-53d6-49ce-817c-5f63e6cad69e.png)
****